scripts/leapfrog/elasticsearch_upgrade.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+set -e -u -x
+set -o pipefail
+
+# Check upgrade flag
+if [[ -n "$UPGRADE_ELASTICSEARCH" ]]; then
+  if [[ "$UPGRADE_ELASTICSEARCH" == "NO" ]]; then
+    echo 'Please make sure you want to upgrade Elasticsearch.'
+    exit
+  fi
+else
+  echo "Please preset env var before leap."
+  exit
+fi
+
+# Update pip.conf on Elasticsearch container
+pushd /opt/rpc-openstack/rpcd/playbooks
+openstack-ansible ../../scripts/leapfrog/playbooks/elasticsearch-postleap-pip-upgrade.yml
```

Comment:
For consistency and readability these commands should all be indented two spaces between the pushd/popd lines.

Reply:
Like?
  pushd xxx
openstack-ansible xxx
  popd 
I can fix it.

Reply:
Cool will add them